### PR TITLE
task lesson N2

### PR DESCRIPTION
**Pull request by Lilit-Lewonyan**
I  at first pushed it without creating per-branch, and as a result, I couldn't add Karen as a reviewer. ( I pushed it directly to my main origin). That's why I added some comments and create a per-branch and pushed the second time.

### CODE REVIEW

File: src/FirstTask.java [hunk 1]
```diff
@@ -1,11 +1,12 @@
...
       for (int i =1; i<=9; i++){
           printline(i);
       }
 
     }
-
+//print method
     private static void printline(int line){
         String string1 = "-";
         String string2 = ". .";
```
Comment (Karengrigoryan060) [inline src/FirstTask.java:12]: are you sure this is working?

**Review by Karengrigoryan060 — COMMENTED** [review]
please, use only functionality, which we have learned during our training.

File: src/FirstTask.java (hunk 1 above)
Comment (Lilit-Lewonyan) [reply src/FirstTask.java:12]: Yes it is working. But I am not sure that it is what you expected.

Comment (Lilit-Lewonyan) [reply src/FirstTask.java:12]: Should I use  only primitive data types, yes?

**Review by Karengrigoryan060 — APPROVED** [review]

**Pull request merged**